images/microshift-bootc.yml

```diff
@@ -0,0 +1,29 @@
+# Disable automated builds, this image is intended to be built manually like microshift rpm
+mode: disabled
+arches:
+- x86_64
+- aarch64
+content:
+  source:
+    dockerfile: packaging/imagemode/Containerfile.repobase
+    git:
+      branch:
+        target: release-{MAJOR}.{MINOR}
+      url: git@github.com:openshift-priv/microshift.git
+      web: https://github.com/openshift/microshift
+    ci_alignment:
+      streams_prs:
+        enabled: false
```

Comment:
Think this should be enabled eventually. Dos that need a tracker?

Reply:
The way https://github.com/openshift/microshift/blob/release-4.18/packaging/imagemode/Containerfile.repobase is setup is different, in that it doesn't have an upstream image that gets used in CI / prowjob. So we don't have an `upstream_image` defined in streams.yml for `bootc-rhel9` and this alignment won't be needed. Giga indicated microshift CI builds are setup differently so they don't exactly need this (is what I understood)

Reply:
I'm not sure I understand this comment. We must have a "MicroShift containerfile bit" in the flow so that MicroShift team controls `only` how the s/w is installed and configurred. 
I'm ready to change the [Containerfile.repobase](https://github.com/openshift/microshift/blob/release-4.18/packaging/imagemode/Containerfile.repobase) file any way necessary to accomodate your pipeline needs, but it has to come from MicroShift repo.

Reply:
Resolving this, will followup on this once we get closer to finalizing pipeline.